src/util/error.js

```diff
@@ -23,8 +23,36 @@ export function getBadUserError(): Error {
 
 export function isUserCorrectableError(status: number) {
     return (
+        !status ||
```

Comment:
This will start showing "client side" errors to the user again, including CONNECTION_RESET, which we've been seeing due to proxy issues. Is this necessary as part of this change?

Reply:
I added it cause `CONNECTION_RESET` is a user correctable action IMO. They could have network issues

Reply:
As discussed, removing this for now